app/database/category.py

```diff
@@ -1,31 +1,35 @@
-from sqlalchemy import Column, Integer, String, Boolean, Sequence, ForeignKey
+from sqlalchemy import Column, Integer, String, Boolean, Sequence, ForeignKey, Date
 
 
-def create_income_category_table(db):
-    class IncomeCategory(db.Model):
-        __tablename__ = "income_category"
+def create_category_table(db):
+    class Category(db.Model):
+        __tablename__ = "category"
         id = Column(Integer, Sequence("seq_income_category_id", start=1), primary_key=True)
-        category_name = Column(String, nullable=False)
+        name = Column(String, nullable=False)
+        description = Column("Description", String)
```

Comment:
I don't think that we need description as the name of category should be self explanatory.